### ISSUE TITLE
Capistrano only outputs it's git:wrapper line when using airbrush

### ISSUE THREAD

**Issue by nesl247** [Shopify/shipit-engine#853]
When using airbrush (the default formatter), only the git:wrapper line is output. After that, it keeps running but nothing shows up.

I'm aware of #529, but we are using the latest release of both capistrano and airbrush.

**Comment by nesl247** [comment]
It's still running actually, but I believe shipit eventually kills the job because it says it hasn't received any output in X amount of minutes.

**Comment by byroot** [comment]
Right, that's what I meant by "stuck".

Just a guess, but usually when this happens it's because capistrano is asking for an input, possibly airbrush is causing this, it's hard to tell without being able to inspect the process. I'd recommend trying the classics, starting with strace.

But overall it's extremely unlikely that it's Shipit causing this. It might be Shipit's PTY that is throwing Airbrake off, but even if that's the case the fix would need to be in Airbrake.

Alternatively you can disable Airbrake when capistrano is invoke on Shipit by looking at `ENV['SHIPIT']`

**Comment by nesl247** [comment]
It is definitely airbrush related, its in the original comment as well. I do not know though if it is airbrush having a but or shipit's interaction with airbrush that has this bug, hence why I posted here as we use airbrush just fine in Jenkins.

**Comment by byroot** [comment]
Right. The possible difference between Shipit and Jenkins is that it tries to act a bit like a terminal using [`PTY.spawn`](https://ruby-doc.org/stdlib-2.2.3/libdoc/pty/rdoc/PTY.html#method-c-spawn) which sometimes trick some programs into thinking they are spawned by a human and can ask for input.

Did you try `strace`? It would make it relatively easy to figure what blocks the process.
